docs/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+
+## [3.4.0] - 2023-08-14
```

Comment:
According to Jira, the following has not been released yet: 787, 334, 1165, 1172

I am unsure about 1115, as it does not have `Fix Version/s` in Jira set.

Reply:
Jira is wrong and the changelog is correct.